spec/string_spec.rb

```diff
@@ -29,4 +31,35 @@
     end
   end
 
+  it "replaces spaces with underscores" do
+    Benchmark.ips do |x|
+      str = Random.new.bytes(1024)
```

Comment:
but, what you're looking for is potential spaces, so anything that generates multiple words.